workerbee/postgresql.py

```diff
@@ -28,257 +34,328 @@ def check_name(table_name):
 SELECT EXISTS(
     SELECT *
     FROM information_schema.tables
-    WHERE table_name = 'wb__{}'
+    WHERE table_name = '{tbl_name}'
 );
-"""
+""".strip()
 
 CREATE_TABLE_QUERY = r"""
-CREATE TABLE wb__{}(
-  job_id TEXT UNIQUE NOT NULL PRIMARY KEY,
-  data json,
-  completed BOOLEAN NOT NULL DEFAULT FALSE,
-  claimed BOOLEAN NOT NULL DEFAULT FALSE
+CREATE TABLE {tbl_name}(
+  id SERIAL PRIMARY KEY,
+  unique_id TEXT UNIQUE,
+  input_data json NOT NULL,
```

Comment:
We could use `JSONB` rather than `JSON`. Main advantage being `JSONB` are equatable, and so, can be held under a unique constraint. Perhaps the schema should be 
```diff 
-  unique_id TEXT UNIQUE,
-  input_data json NOT NULL,
+  input_data JSONB NOT NULL UNIQUE,
-  output_data json,
+  output_data JSONB,
```
Then we can just drop the `unique_id` altogether here. Support is Postgres 9.4+, which I think is acceptable?

Argument: workerbee functions really should be deterministic/idempotent, so duplicate job payloads in a single experiment never make sense.